Dfe.Data.SearchPrototype/Infrastructure/CognitiveSearchServiceAdapter.cs

```diff
@@ -102,7 +103,7 @@ await _searchByKeywordService.SearchAsync<TSearchResult>(
         {
             Establishments = _searchResultMapper.MapFrom(searchResults.Value.GetResults()),
             Facets = searchResults.Value.Facets != null
-                ? _facetsMapper.MapFrom(searchResults.Value.Facets.ToDictionary())
+                ? _facetsMapper.MapFrom(searchResults.Value.Facets.OrderBy(facet => facet.Key).ToDictionary())
```

Comment:
Is sort order/sequencing definitely a search prototype concern?

My worry is that this is being implemented too "low level" (as opposed to, for example, treating it as a display/web UI concern and doing the sort at the last-reasonable-moment within the view code).

Reply:
Roger, I've put the background to this PR in the description, which we should have done in the first place. Maybe it will answer your concern

Reply:
> Maybe it will answer your concern

I've just had a read and I'm not sure it does. 

To illustrate the alternative, what if this sorting was performed in the web mvc project (i.e., the project which displays these values in HTML) as opposed to this shared library project?